packages/forms-web-app/src/pages/projects/project-updates/controller.js

```diff
@@ -0,0 +1,23 @@
+const logger = require('../../../lib/logger');
+const { getProjectUpdatesData } = require('./services');
+const { getPageData } = require('./utils/get-page-data');
+
+const getProjectUpdatesController = async (req, res, next) => {
+	try {
+		const { params } = req;
+		const { case_ref } = params;
+		const { locals } = res;
```

Comment:
since you're using the middleware in the router, it makes the `caseRef` available in locals